projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -291,77 +376,88 @@ abstract function get_connection_id( $connection );
 	 * @return array Connection Meta
 	 */
-	abstract function get_connection_meta( $connection );
+	abstract public function get_connection_meta( $connection );
 
 	/**
 	 * Disconnect a Connection
 	 *
-	 * @param string $service_name 'facebook', 'twitter', etc.
-	 * @param string $connection_id Connection ID
-	 * @param false|int $_blog_id The blog ID. Use false (default) for the current blog
-	 * @param false|int $_user_id The user ID. Use false (default) for the current user
-	 * @param bool $force_delete Whether to skip permissions checks
+	 * @param string    $service_name 'facebook', 'twitter', etc.
+	 * @param string    $connection_id Connection ID.
+	 * @param false|int $_blog_id The blog ID. Use false (default) for the current blog.
+	 * @param false|int $_user_id The user ID. Use false (default) for the current user.
+	 * @param bool      $force_delete Whether to skip permissions checks.
 	 * @return false|void False on failure. Void on success.
 	 */
-	abstract function disconnect( $service_name, $connection_id, $_blog_id = false, $_user_id = false, $force_delete = false );
+	abstract public function disconnect( $service_name, $connection_id, $_blog_id = false, $_user_id = false, $force_delete = false );
 
 	/**
 	 * Globalizes a Connection
 	 *
-	 * @param string $connection_id Connection ID
+	 * @param string $connection_id Connection ID.
 	 * @return bool Falsey on failure. Truthy on success.
 	 */
-	abstract function globalize_connection( $connection_id );
+	abstract public function globalize_connection( $connection_id );
 
 	/**
 	 * Unglobalizes a Connection
 	 *
-	 * @param string $connection_id Connection ID
+	 * @param string $connection_id Connection ID.
 	 * @return bool Falsey on failure. Truthy on success.
 	 */
-	abstract function unglobalize_connection( $connection_id );
+	abstract public function unglobalize_connection( $connection_id );
 
 	/**
 	 * Returns an external URL to the Connection's profile
 	 *
-	 * @param string $service_name 'facebook', 'twitter', etc.
-	 * @param object|array The Connection object (WordPress.com) or array (Jetpack)
+	 * @param string       $service_name 'facebook', 'twitter', etc.
+	 * @param object|array $connection The Connection object (WordPress.com) or array (Jetpack).
 	 * @return false|string False on failure. URL on success.
 	 */
-	function get_profile_link( $service_name, $connection ) {
+	public function get_profile_link( $service_name, $connection ) {
 		$cmeta = $this->get_connection_meta( $connection );
 
 		if ( isset( $cmeta['connection_data']['meta']['link'] ) ) {
-			if ( 'facebook' == $service_name && 0 === strpos( wp_parse_url( $cmeta['connection_data']['meta']['link'], PHP_URL_PATH ), '/app_scoped_user_id/' ) ) {
-				// App-scoped Facebook user IDs are not usable profile links
+			if ( 'facebook' === $service_name && 0 === strpos( wp_parse_url( $cmeta['connection_data']['meta']['link'], PHP_URL_PATH ), '/app_scoped_user_id/' ) ) {
```

Comment:
Nitpick: something scary going on here, maybe worth converting `0 === strpos( wp_parse_url( $cmeta['connection_data']['meta']['link'], PHP_URL_PATH ), '/app_scoped_user_id/' )` to a method of its own which will make it easier to read?

Reply:
I agree that this is a bit ugly, but as this is the only occurrence, and we're focusing on the code sniffs, I think it's fine to leave it for now.